x-pack/plugins/rule_registry/server/routes/get_alert_by_id.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IRouter } from 'kibana/server';
+import * as t from 'io-ts';
+import { id as _id } from '@kbn/securitysolution-io-ts-list-types';
+import { transformError } from '@kbn/securitysolution-es-utils';
+
+import { RacRequestHandlerContext } from '../types';
+import { BASE_RAC_ALERTS_API_PATH } from '../../common/constants';
+import { buildRouteValidation } from './utils/route_validation';
+
+export const getAlertByIdRoute = (router: IRouter<RacRequestHandlerContext>) => {
+  router.get(
+    {
+      path: BASE_RAC_ALERTS_API_PATH,
+      validate: {
+        query: buildRouteValidation(
+          t.intersection([
+            t.exact(
+              t.type({
+                id: _id,
+              })
+            ),
+            t.exact(
+              t.partial({
+                indexName: t.string,
+              })
+            ),
+          ])
+        ),
+      },
+      options: {
+        tags: ['access:rac'],
+      },
+    },
+    async (context, request, response) => {
+      try {
+        const alertsClient = await context.rac.getAlertsClient();
+        const { id, indexName } = request.query;
+        const alert = await alertsClient.get({ id, index: indexName });
+        return response.ok({
+          body: alert,
+        });
+      } catch (exc) {
+        const err = transformError(exc);
+        const contentType = {
+          'content-type': 'application/json',
+        };
+        const defaultedHeaders = {
+          ...contentType,
+        };
+
+        return response.custom({
```

Comment:
Any reason you're using `response.custom` instead of [`response.customError`](https://github.com/legrego/kibana/blob/48523e5066fae3bd769a064ea78a08a9e545331e/src/core/server/http/router/response.ts#L184-L200)?

Reply:
No particular reason. Code was borrowed from elsewhere. Updated to use `customError`. 

https://github.com/elastic/kibana/pull/100705/commits/8d320376a54f21ec5f104669084a1c501f060e60#diff-4f5192cdb9eeb36bfa158b259abd5258c9c453f419e3f06c5cd897d8d3d76991R58